Core/AppRuntime/Source/AppRuntime.cpp

```diff
@@ -36,6 +37,26 @@ namespace Babylon
         m_workQueue->Resume();
     }
 
+    std::string getStringPropertyFromError(Napi::Error error, const char* propertyName)
+    {
+        Napi::Value value = error.Get(propertyName);
+        if (value.IsUndefined())
+        {
+            return "";
+        }
+        return value.ToString().Utf8Value();
+    }
+
+    int32_t getNumberPropertyFromError(Napi::Error error, const char* propertyName)
+    {
+        Napi::Value value = error.Get(propertyName);
+        if (value.IsUndefined())
+        {
+            return -1;
+        }
+        return value.ToNumber().Int32Value();
+    }
+
```

Comment:
These should be functions in an anonymous namespace at the top since they don't need access to class members. Also should be UpperCamelCase.

Reply:
Fixed.